modules/photo/src/ccm/utils.hpp

```diff
@@ -25,7 +25,7 @@ double gammaCorrection_(double element, double gamma);
     @param gamma a constant for gamma correction.
     @param dst the output array, type of Mat.
  */
-Mat gammaCorrection(const Mat& src, double gamma, Mat dst=Mat());
+ CV_EXPORTS void gammaCorrection(InputArray src, OutputArray dst, double gamma);
```

Comment:
It should go to make ccm header to be included into documentation and bindings. Please CV_EXPORTS_W to generate Java and Python bindings for it too.